docs/understanding-commands.rst

```diff
@@ -0,0 +1,176 @@
+Understanding Toolchain Commands
+=================================
+
+This section provides valuable information on how each of the commands used to compile and build 
+designs in Symbiflow work. It is especially helpful for debugging or for using methods 
+other than a makefile to build your designs, such as a bash or python script. 
+
+The following describes the commands for running each of the steps for a full design flow 
+(synthesis, place and route, and generate bitstream) as well as giving a description of the most
+common flags for those commands. If you would like a more detailed break down of how the design 
+flow for Symbiflow works take a look at the 
+`FPGA Design Flow page <https://symbiflow.readthedocs.io/en/latest/toolchain-desc/design-flow.html>`_.
+
+.. note::
+
+    Files created by the synthesis, implementation, and bitstream generation will be dumped into 
+    the directory from which the command is run by default. To keep all of the files generated by 
+    the toolchain separate from your design files, you might consider running the toolchain 
+    commands in a separate directory from your design files. 
+
+
+
+Synthesis
+----------
+
+To synthesize your designs run the ``symbiflow_synth`` command. The command has the following 
+flags:
+
+.. table:: symbiflow_synth
+
+    +------+---------------------------------------------------------------+
+    | Flag |                            Argument                           |
+    +======+===============================================================+
+    | -t   | Defines the name for the top level module                     |
+    +------+---------------------------------------------------------------+
+    | -v   | A list of paths to verilog/systemVerilog files for the design |
```

Comment:
I think it is better to avoid specifying an unsupported and untested HDL
```suggestion
    | -v   | A list of paths to verilog files for the design |
```